release/kdm/rke2_charts.go

```diff
@@ -0,0 +1,103 @@
+package kdm
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"gopkg.in/yaml.v2"
```

Comment:
this package was archived and is unmaintained, I think you should use https://pkg.go.dev/sigs.k8s.io/yaml instead

Reply:
Done, I also updated other references from `"gopkg.in/yaml.v*"` to `"sigs.k8s.io/yaml"`